### PR TITLE
Microverse Exercise : add homepage

### PR DESCRIPTION
**Pull request by AtillaTahak**
### Microverse Exercise 
- add index.html
- add linters
- add css
- fixed readme
- add test.md

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @AtillaTahak, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] Always capitalize your commit messages. It is a standard practice.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
```
Comment (mke2111) [inline .github/workflows/linters.yml:8]: - [ ] You are having failing linters, Linter result should be green. AKA: Linter should report NO errors.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (mke2111) [inline test.md:1]: - [ ] Unfortunately, the test.md file still exists in your repo. Ensure that it doesn't exist anywhere except being listed in the ignore file.

File: README.md [hunk 3]
```diff
@@ -8,68 +8,37 @@
...
 Additional description about the project and its features.
 
-## Built With
+## Built With 
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
```
Comment (mke2111) [inline README.md:15]: - [ ] Add a more meaningful project title. You are currently having `Project Name` as your title.

Comment (AtillaTahak) [reply README.md]: Thank you for review

**Review by carlos-ssh — APPROVED** [review]
Hi @AtillaTahak , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**